src/backend/PhotoAPI/BusinessLogic/Services/PhotoService.cs

```diff
@@ -30,6 +31,14 @@ public async Task<IEnumerable<PhotoListDTO>> GetPhotosAsync(string userId)
             return userPhotos.OrderByDescending(p => p.UploadDate).Select(_mapper.Map<PhotoListDTO>);
         }
 
+        public async Task<PhotoViewDTO> GetPhotoOrDefaultAsync(Guid photoId)
+        {
+            PhotoDocument userPhoto = await _elasticService.GetPhotoOrDefaultAsync(photoId);
+            if (userPhoto == null) return null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4949566)